### DISCUSSION TITLE
Could not pnpm check package.json engines config ?

### DISCUSSION THREAD

**Discussion by tarunama** [pnpm/pnpm#4884] [Q&A] [answered]
`npm install` command shows below warning.

```bash
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@sveltejs/kit@1.0.0-next.350',
npm WARN EBADENGINE   required: { node: '>=16.7' },
npm WARN EBADENGINE   current: { node: 'v14.17.0', npm: '8.5.5' }
npm WARN EBADENGINE }
```

But, `pnpm install` command does not show warning.
This is valid behavior ?

**Comment by mcmxcdev** [comment, marked as answer]
You need to set `engine-strict=true` in your `.npmrc` if you want pnpm to fail on invalid engines declared in `package.json`

See more: https://pnpm.io/npmrc#engine-strict
